packages/protocol/contracts/L1/gov/TaikoTimelockController.sol

```diff
@@ -19,8 +19,6 @@ import
 import "../../common/OwnerUUPSUpgradable.sol";
 
 contract TaikoTimelockController is OwnerUUPSUpgradable, TimelockControllerUpgradeable {
-    uint256[50] private __gap;
```

Comment:
Because `__gap` is usually needed to reserve storage space (and mainly in parent contracts), but here there is no storage variables in this contract (yet) so we can simply add new storage variables any time we upgrade.

Reply:
If you remove `__gap` here, the sub-contract's storage layout will be totally shifted.

A contract having no storage doesn't mean it won't have storage variable in the future. We need to keep it.

Reply:
Dani, you need to revert this change.